libs/ledger-live-common/src/exchange/swap/index.ts

```diff
@@ -100,7 +100,7 @@ const swapProviders: Record<string, SwapProviderConfig> = {
 };
 
 const getProviderConfig = (providerName: string): SwapProviderConfig => {
-  const res = swapProviders[providerName.toLowerCase()];
+  const res = swapProviders[providerName.toLowerCase()] || {};
```

Comment:
This is error prone and also not really necessary once you rework / break down your `Rate` component since the DEXs should not require / need registration nor KYC

Reply:
Any logic has been removed from LLC